source/containers.tex

```diff
@@ -3233,15 +3233,10 @@
 
 \pnum
 \effects
-Equivalent to \tcode{a.emplace(std::forward<Args>(args)...)},
+Equivalent to: \tcode{return a.emplace(std::forward<Args>(args)...).first},
```

Comment:
I've just realised that this doesn't work, because `a_uniq.emplace` and `a_eq.emplace` return different things. It's not a pair for `a_eq`, so you can't do `.first`

So maybe what you had before was better, sorry.

Reply:
I must not have been paying attention either; my bad. The latest commit should resolve everything.